ui/app/styles/core/message.scss

```diff
@@ -119,6 +119,15 @@
   &.has-top {
     margin-top: 1rem;
   }
+
+  &.size-small {
+    padding-bottom: $size-8;
```

Comment:
Mixing the padding into the classname `size-small` feels a bit conflated to me. I wonder if instead we should keep all padding/margins out of this component, and instead wrap with a div that has the padding? especially since there's no background color here, it should probably be a margin anyway.